llvm/lib/Analysis/InlineOrder.cpp

```diff
@@ -114,7 +114,10 @@ class CostBenefitPriority {
   CostBenefitPriority(const CallBase *CB, FunctionAnalysisManager &FAM,
                       const InlineParams &Params) {
     auto IC = getInlineCostWrapper(const_cast<CallBase &>(*CB), FAM, Params);
-    Cost = IC.getCost();
+    if (IC.isVariable())
```

Comment:
Does it always need to check `isVariable` for the return of `getInlineCostWrapper`? As far as I can see, there are three usages of `getInlineCostWrapper` and all needs those checks, maybe we can just move it into `getInlineCostWrapper`?

Reply:
Oh, I think it needs to use the `IC` not only the cost, (I was just thinking to avoid those three duplicated code), no better idea so far, feel free to ignore.